doc/sphinx/addendum/type-classes.rst

```diff
@@ -381,37 +371,65 @@ Summary of the commands
 Besides the :cmd:`Class` and :cmd:`Instance` vernacular commands, there are a
 few other commands related to typeclasses.
```

Comment:
Just delete it?  The wording is more for a style in which the documentation is read sequentially rather than jumping to the specific thing the reader is looking for--as is common in a reference work.

Reply:
Yes, deleting it is good.

Reply:
I indented `Existing Instance`.  What other command(s) do you want me to indent?

Reply:
Looks good as is.